File: motor3d 2.0.ConvertedToC#/ObjetoPoliedrico.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using System.Linq;
using System.Xml.Linq;
using Motor3D.Espacio3D;

namespace Motor3D.Primitivas3D
{
	public class ObjetoPrimitiva3D : ObjetoGeometrico3D
	{

		public new event ModificadoEventHandler Modificado;
		public new delegate void ModificadoEventHandler(ref ObjetoPrimitiva3D Sender);

		public ObjetoPrimitiva3D() : base()
		{
		}

		public override string ToString()
		{
			return base.ToString();
		}
	}
}

File: motor3d 2.0/objetopoliedrico.vb
Imports Motor3D.Espacio3D

Namespace Primitivas3D
    Public Class ObjetoPrimitiva3D
        Inherits ObjetoGeometrico3D

        Public Shadows Event Modificado(ByRef Sender As ObjetoPrimitiva3D)

        Public Sub New()
            MyBase.New()
        End Sub

        Public Overrides Function ToString() As String
            Return MyBase.ToString()
        End Function
    End Class
End Namespace

